packages/react-components/components/Button.v2.tsx

```diff
@@ -27,7 +27,7 @@ export enum BtnSizes {
 export interface ButtonProps {
   onPress: () => void
   style?: StyleProp<ViewStyle>
-  text: string
+  text: string | JSX.Element
```

Comment:
This is needed, when we show <CurrencyDisplay /> inside the button text.

Reply:
It's preferable to use `React.ReactNode` 
https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement